characters/blaise.json

```diff
@@ -0,0 +1,13 @@
+{
+    "sentences": [
+        "Faudrait voir à pas oublier que sans les Romains, on serait jamais qu’une bande de pécores qui pataugent dans la bouse !",
+
+        "Oh oui oui, c'est magnifique, oui, oui, il y a des taches, c'est tout barré, j'ai tellement raturé que j'ai transpercé le papier...C'est immonde...on dirait que j'ai lavé par terre avec, mais c'est fait...",
+        
+        "Voilà ! C’est tout ce qu’y a ! Unisson, quarte, quinte et c’est marre ! Tous les autres intervalles, c’est de la merde ! Le prochain que je chope en train de siffler un intervalle païen, je fais un rapport au pape !",
+        
+        "Vous racontez ce qui s'est passé, d'un coup, sans vous arrêter, et si vous changez des trucs, je vous envoie le registre à travers la gueule. Vu ?",
+        
+        "Ouais, ben si vous vouliez du captivant, fallait peut-être me faire lire autre chose que des textes de lois !"
+    ]

```

Comment:
you may add :

- Non mais attendez, je crois qu'on s'est mal compris, là : vous avez une idée du temps qu'il me faut pour tracer une lettre avec ces PUTAINS DE PLUMES ?!
- Non, je suis désolé, moi je peux pas faire une légende avec un fakir qui enroule sa bite autour d'un bâton. En plus je sais même pas comment ça se dit en latin.
- MAIS VOUS ÊTES DES MALADES !!!
- En voilà une bonne chose, faisons la liste des cons, et puis ça me donnera l'occasion de gratter du papier.

Reply:
Added ;)